src/snowflake/connector/log_configuration.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+
+from __future__ import annotations
+
+import logging
+import os
+from datetime import datetime
+
+from snowflake.connector.config_manager import CONFIG_MANAGER
+from snowflake.connector.constants import CONNECTIONS_FILE
+from snowflake.connector.secret_detector import SecretDetector
+
+
+class EasyLoggingConfigPython:
+    def __init__(self):
+        self.log_path = None
+        self.log_level = None
+        self.log_file_name = None
+        self.parse_config_file()
+
+    def parse_config_file(self):
+        CONFIG_MANAGER.read_config()
+        data = CONFIG_MANAGER["connections"]
+        if "common" in data:
+            if "log_level" not in data["common"] or "log_path" not in data["common"]:
+                raise ValueError(
+                    f"config file at {CONNECTIONS_FILE} is not in correct form, please verify your config file"
+                )
+            data = data["common"]
```

Comment:
How about something like this for a more efficient approach?  (Keep in mind `:=` operator was added in Python 3.8).

```python
if common := data.get('common'):
    self.log_level = data.get('level')
    self.path = data.get('path')
    self.save_logs = data.get('save_logs')
    # ... do value sanity checking
```